Services/UI/OJS.Services.Ui.Models/Contests/ValidationModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace OJS.Services.Ui.Models.Contests;
+
+public class ValidationModel
```

Comment:
This is too generic a name. It must be more descriptive

Reply:
Renamed to ContestValidationModel.